static/server/taskbar.css

```diff
@@ -37,14 +40,22 @@ li.ui-state-focus {
 	text-decoration: underline;
 }
 
-@media only screen and (min--moz-device-pixel-ratio: 1.5),
-only screen and (-o-min-device-pixel-ratio: 1.5/1),
-only screen and (-webkit-min-device-pixel-ratio: 1.5),
-only screen and (min-device-pixel-ratio: 1.5) {
-    #kiwixtoolbar button, #kiwixtoolbar input {
-    	font-size: 2em;
+/* the breakpoint "600px" is the width of the 3 btns "library", "home" and "randoml" */
+@media (max-width: 600px) {
+    .kiwixsearch {
+        float: none;
+        margin-top: 5px;
     }
+
     #kiwixsearchbox {
-		width: 7em;
-	}
```

Comment:
Theses lines are meant to increase font size and increase the size of search bar for high density pixel screens.
I handle the size of the search bar differently.
But the 2em font size break my design.
I dont think it is that important to increase the font size because high density devices already handle it themselves. It usually is considered a bad practice to handle the font size in CSS for different pixel ratio.